### ISSUE TITLE
The results are not consisstent

### ISSUE THREAD

**Issue by chouqin3** [sud0301/semisup-semseg#12]
I run the command 
`--dataset pascal_voc  --checkpoint-dir ./checkpoints/voc_semi_0_125 --labeled-ratio 0.125  --ignore-label 255  --num-classes 21`
and achieve  64.11%, that paper says 65.4%
![image](https://user-images.githubusercontent.com/12481496/104455991-ff5a2600-55e2-11eb-8e69-e345e1cd77a6.png)
`--dataset pascal_voc  --checkpoint-dir ./checkpoints/voc_semi_0_02 --labeled-ratio 0.02  --ignore-label 255  --num-classes 21`
and achieve  53.59%, that paper says 58.1%, what's the problems?
![image](https://user-images.githubusercontent.com/12481496/104456565-d2f2d980-55e3-11eb-920a-78c895d46ec1.png)

**Comment by sud0301** [comment]
Hi, 
Thank you for your interest in the work.
Please verify the labeled-ratio used in the command-line arguments. According to your current arguments of labeled-ratio=2%, the expected performance is 58.1% (Table1 without COCO pre-training ) as mentioned in the paper.

**Comment by chouqin3** [comment]
I've updated the questions, thanks for your attention, could you explain it?

**Comment by sud0301** [comment]
1. One of the possible reason for the discrepancy in the results could be due to the usage of different splits. Try to run using different splits and average. The reported performance is averaged over 3 random splits. 
2. You should evaluate the validation performance at every 5k iterations. The optimal performance for smaller splits (for eg. with 5% labels) might occur earlier than 35k iterations.
